packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -719,6 +719,46 @@ describe('PerformanceMonitoring', function() {
       expect(newTr.name).toBe('Click - button["purchase"]')
     })
 
+    it('should create click transactions on window', () => {
+      let transactionService = performanceMonitoring._transactionService
+      let etsub = performanceMonitoring.getEventTargetSub()
+      patchEventHandler.observe(EVENT_TARGET, (event, task) => {
+        etsub(event, task)
+      })
+
+      const listener = e => {
+        expect(e.type).toBe('click')
+      }
+
+      window.addEventListener('click', listener)
+      document.body.click()
+
+      let tr = transactionService.getCurrentTransaction()
+      expect(tr).toBeDefined()
+      expect(tr.name).toBe('Click - html')
```

Comment:
shouldn't this be `BODY`?

Reply:
@vigneshshanmugam `HTML` is parentNode of `BODY`, so I think `HTML` is better than `BODY` when target is `window` or `document`.